site/content/docs/main/file-system-backup.md

```diff
@@ -0,0 +1,628 @@
+storage providers are include here [supported providers](supported-providers.md), for which, Velero pre-defines the endpoints; if you 
+want to use a different backup storage, make sure it is S3 compatible and you provide the correct bucket name and endpoint in 
+BackupStorageLocation. Alternatively, for Restic, you could set the `resticRepoPrefix` value in BackupStorageLocation. For example, 
+on AWS, `resticRepoPrefix` is something like `s3:s3-us-west-2.amazonaws.com/bucket` (note that `resticRepoPrefix` doesn't work for Kopia). 
+Velero handles the creation of the backup repo prefix in the backup storage, so make sure it is specified in BackupStorageLocation correctly.  
+
+Velero creates one backup repo per namespace. For example, if backing up 2 namespaces, namespace1 and namespace2, using kopia 
+repository on AWS S3, the full backup repo path for namespace1 would be `https://s3-us-west-2.amazonaws.com/bucket/kopia/ns1` and 
+for namespace2 would be `https://s3-us-west-2.amazonaws.com/bucket/kopia/ns2`.  
+
+There may be additional installation steps depending on the cloud provider plugin you are using. You should refer to the 
+[plugin specific documentation](supported-providers.md) for the must up to date information.  
+
+### Configure Node Agent DaemonSet spec
+
+After installation, some PaaS/CaaS platforms based on Kubernetes also require modifications the node-agent DaemonSet spec. 
+The steps in this section are only needed if you are installing on RancherOS, OpenShift, VMware Tanzu Kubernetes Grid 
+Integrated Edition (formerly VMware Enterprise PKS), or Microsoft Azure.  
+
+
+**RancherOS**
+
+
+Update the host path for volumes in the nonde-agent DaemonSet in the Velero namespace from `/var/lib/kubelet/pods` to 
+`/opt/rke/var/lib/kubelet/pods`.  
+
+```yaml
+hostPath:
+  path: /var/lib/kubelet/pods
+```
+
+to
+
+```yaml
+hostPath:
+  path: /opt/rke/var/lib/kubelet/pods
+```
+
+
+**OpenShift**
+
+
+To mount the correct hostpath to pods volumes, run the node-agent pod in `privileged` mode.
+
+1. Add the `velero` ServiceAccount to the `privileged` SCC:
+
+    ```
+    $ oc adm policy add-scc-to-user privileged -z velero -n velero
+    ```
+
+2. For OpenShift version  >= `4.1`, modify the DaemonSet yaml to request a privileged mode:
+
+    ```diff
+    @@ -67,3 +67,5 @@ spec:
+                  value: /credentials/cloud
+                - name: VELERO_SCRATCH_DIR
+                  value: /scratch
+    +          securityContext:
+    +            privileged: true
+    ```
+
+    or
+
+    ```shell
+    oc patch ds/node-agent \
+      --namespace velero \
+      --type json \
+      -p '[{"op":"add","path":"/spec/template/spec/containers/0/securityContext","value": { "privileged": true}}]'
+    ```
+
+3. For OpenShift version  < `4.1`, modify the DaemonSet yaml to request a privileged mode and mount the correct hostpath to pods volumes.
```

Comment:
You can remove this entire section. Since Velero now only supports Kubernetes 1.16 and later, We don't need to worry about OpenShift older than 4.1 anymore, since OpenShift 4.1 is based on Kubernetes 1.13.

Reply:
Done, thanks.